.github/workflows/ci-all-in-one-build.yml

```diff
@@ -51,6 +51,10 @@ jobs:
       if: github.ref_name != 'main'
       run: bash scripts/build-all-in-one-image.sh pr-only
 
+    - name: Build and test jaeger-v2 as all-in-one
+      if: github.ref_name != 'main'
+      run: BINARY=jaeger-v2 bash scripts/build-all-in-one-image.sh pr-only
```

Comment:
this only added 39s to the total run, I think it's worth keeping in this workflow for now